src/pudl/metadata/codes.py

```diff
@@ -1231,4 +1231,256 @@
+                ("AZPS", "arizona_public_service", "Arizona Public Service Company"),
+                (
+                    "BANC",
+                    "northern_california",
+                    "Balancing Authority of Northern California",
+                ),
+                ("BPAT", "bonneville_power", "Bonneville Power Administration"),
+                (
+                    "CHPD",
+                    "public_utility_district_of_chelan_county",
+                    "Public Utility District No. 1 of Chelan County",
+                ),
+                (
+                    "CISO",
+                    "california_iso",
+                    "California Independent System Operator",
+                ),
+                ("CPLE", "duke_energy_progress_east", "Duke Energy Progress East"),
+                ("CPLW", "duke_energy_progress_west", "Duke Energy Progress West"),
+                (
+                    "CSTO",
+                    "constellation",
+                    "Constellation Energy Control and Dispatch, LLC",
+                ),
+                (
+                    "CSWS",
+                    "public_service_company_of_oklahoma_and_southwestern_electric",
+                    "American Electric Power Service Corp. As Agent For Public Svc. Co. Of Oklahoma & SW Ele Pwr Co.",
+                ),
+                ("DEAA", "arlington_valley", "Arlington Valley, LLC - AVBA"),
+                (
+                    "DOPD",
+                    "public_utility_of_douglas_county",
+                    "PUD No. 1 of Douglas County",
+                ),
+                ("DUK", "duke_energy_carolinas", "Duke Energy Carolinas"),
+                ("EDE", "empire_district", "The Empire District Electric Company"),
+                ("EEI", "electric_energy", "Electric Energy, Inc."),
+                ("EPE", "el_paso", "El Paso Electric Company"),
+                (
+                    "ERCO",
+                    "electric_reliability_council_of_texas",
+                    "Electric Reliability Council of Texas, Inc.",
+                ),
+                (
+                    "FMPP",
+                    "florida_municipal_power_pool",
+                    "Florida Municipal Power Pool",
+                ),
+                ("FPC", "progress_energy_florida", "Progress Energy Florida"),
+                ("FPL", "florida_power_and_light", "Florida Power & Light Company"),
+                (
+                    "GCPD",
+                    "public_utility_grant_county",
+                    "Public Utility District No. 2 of Grant County, Washington",
+                ),
+                ("GRDA", "grand_river_dam", "Grand River Dam Authority"),
+                ("GRIF", "griffith_energy", "Griffith Energy, LLC"),
+                ("GRIS", "gridforce_south", "Gridforce South"),
+                ("GRMA", "gila_river_power", "Gila River Power, LLC"),
+                ("GVL", "gainsville_regional", "Gainesville Regional Utilities"),
```

Comment:
Spelling mistake in the label -- missing the E in gainesville

Reply:
good catch!